packages/react-baseline-inputs/src/types.ts

```diff
@@ -95,6 +120,18 @@ export interface FieldProps extends FieldInputProps {
   render: (props: object) => React.ReactNode;
 }
 
+export interface FieldSetProps extends HTMLProps<HTMLFieldSetElement> {
+  legend: React.ReactNode;
```

Comment:
We probably need to be able to disable the legend in order to support `wrapper={false}`, so maybe this should be optional.